src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./Footer.css";
+import { ReactComponent as LinkedIn } from "../../assets/linkedin-icon.svg";
+import { ReactComponent as GitHub } from "../../assets/github-icon.svg";
+
+function Footer() {
+  return (
+    <nav className="wrapper" id="footer" data-testid="footer-wrapper">
+      <a href="http://www.linkedin.com/in/luydmila-kolasava-1940a01b0">
```

Comment:
please add ability to open links in the new tab

Reply:
done